src/platform-implementation-js/driver-common/sidebar/constants.ts

```diff
@@ -0,0 +1,27 @@
+import cx from 'classnames';
+import { legacyIdMap } from '../../lib/idMap';
+
+const cssGlobal = 'inboxsdk__app_sidebar_waiting_platform';
+
+const enum SidebarClassName {
+  /** v1.1.0 CSS module selector, */
+  v1_1_0 = 'inboxsdk__a0oshmeH6N_2vLtc3DNT',
+  /** v1.1.1 CSS module selector, */
+  v1_1_1 = 'inboxsdk__BV_FA4qVT4qYxs3LbEU4',
+}
+
+export const sidebarWaitingPlatformSelector = `.${cssGlobal},
+  ${[SidebarClassName.v1_1_0, SidebarClassName.v1_1_1]
+    .map((x) => '.' + x)
+    .join(', ')},
+  .${
+    // Pre CSS module selector <1.1.0
+    legacyIdMap('app_sidebar_waiting_platform')
+  }` as const;
```

Comment:
It's weird how this declaration uses a mix of template strings and array mapping to create its result. Here's a more consistent version:

```suggestion
export const sidebarWaitingPlatformSelector = [
  cssGlobal,
  SidebarClassName.v1_1_0,
  SidebarClassName.v1_1_1,
  // Pre CSS module selector <1.1.0
  legacyIdMap('app_sidebar_waiting_platform'),
]
    .map((x) => '.' + x)
    .join(', ');